configure.ac

```diff
@@ -4,9 +4,10 @@ define(_CLIENT_VERSION_MAJOR, 0)
 define(_CLIENT_VERSION_MINOR, 11)
 define(_CLIENT_VERSION_REVISION, 2)
 define(_CLIENT_VERSION_BUILD, 0)
+define(_CLIENT_VERSION_ZCASH, 3)
 define(_CLIENT_VERSION_IS_RELEASE, true)
 define(_COPYRIGHT_YEAR, 2015)
-AC_INIT([Bitcoin Core],[_CLIENT_VERSION_MAJOR._CLIENT_VERSION_MINOR._CLIENT_VERSION_REVISION],[https://github.com/bitcoin/bitcoin/issues],[bitcoin])
+AC_INIT([Bitcoin Core],[_CLIENT_VERSION_MAJOR._CLIENT_VERSION_MINOR._CLIENT_VERSION_REVISION.z_CLIENT_VERSION_ZCASH],[https://github.com/bitcoin/bitcoin/issues],[bitcoin])
```

Comment:
I don't know what context the ``Bitcoin Core`` label is used in, but let's go ahead and replace it with ``Zcash`` and if anything breaks from that, fix it in follow up PRs.